### DISCUSSION TITLE
Addition of SECURITY.md?

### DISCUSSION THREAD

**Discussion by andy778** [usebruno/bruno#3703] [Ideas]
Would be good to know on first page on how the project handles life cycle and security bugs. 
Definition from scorecard is good start https://github.com/ossf/scorecard/blob/main/docs/checks.md#security-policy or https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository

**Comment by pieterocp** [comment]
Yeah, could be as simple as email `security@usebruno.com` and we'll get back to you.

**Reply by pieterocp** [reply]
Looks like it's happening with https://github.com/usebruno/bruno/pull/4266, how exciting!

**Reply by pieterocp** [reply]
https://github.com/usebruno/bruno/blame/main/security.md

Looks to be there, think this begins to cover it, so might be able to be closed?

**Comment by helloanoop** [comment]
[security.md](https://github.com/usebruno/bruno/blame/main/security.md) has been added to repo
